README.rst

```diff
@@ -1 +1,142 @@
-# aiohttp-csrf
\ No newline at end of file
+aiohttp_csrf
+============
+
+The library provides csrf (xsrf) protection for `aiohttp.web`__.
+
+.. _aiohttp_web: https://aiohttp.readthedocs.io/en/latest/web.html
+
+__ aiohttp_web_
+
+.. image:: https://img.shields.io/travis/wikibusiness/aiohttp-csrf.svg
+    :target: https://travis-ci.org/wikibusiness/aiohttp-csrf
+
+Usage
+-----
+
+The library allows us to implement csrf (xsrf) protection for requests
+
+Basic usage example:
+
+.. code-block:: python
+
+    import aiohttp_csrf
+    from aiohttp import web
+
+    FORM_FIELD_NAME = '_csrf_token'
+    COOKIE_NAME = 'csrf_token'
+
+
+    async def handler_get_form_with_token(request):
+        token = await aiohttp_csrf.generate_token(request)
+
+        body = '''
+            <html>
+                <head><title>Form with csrf protection</title></head>
+                <body>
+                    <form method="POST" action="/post_with_check">
+                        <input type="hidden" name="{field}" value="{value}" />
+                        <input type="text" name="name" />
+                        <input type="submit" value="Say hello">
+                    </form>
+                </body>
+            </html>
+        '''
+
+        body = body.format(
+            field=FORM_FIELD_NAME,
+            value=token,
+        )
+
+        return web.Response(body=body.encode('utf-8'), content_type='text/html')
+
+
+    async def handler_get_form_without_token(request):
+        body = '''
+            <html>
+                <head><title>Form without csrf protection</title></head>
+                <body>
+                    <form method="POST" action="/post_without_check">
+                        <input type="text" name="name" />
+                        <input type="submit" value="Say hello">
+                    </form>
+                </body>
+            </html>
+        '''
+
+        return web.Response(body=body.encode('utf-8'), content_type='text/html')
+
+
+    async def handler_post_check(request):
+        post = await request.post()
+
+        body = 'Hello, {name}'.format(name=post['name'])
+
+        return web.Response(body=body.encode('utf-8'), content_type='text/html')
+
+
+    @aiohttp_csrf.not_check
```

Comment:
what about to copy names from https://docs.djangoproject.com/en/1.11/ref/csrf/#module-django.views.decorators.csrf 
`csrf_protect`  and `csrf_exempt` ?